HHS/Head-Start-TTADP
# Add Next Steps Page

**Description of change**
Add in Next Steps page for the frontend.
Also looks like a follow up to #222 (for `bin/maintenance`)

**How to test**
Pull down changes, and start server locally
Navigate to the Next Steps and you should be able to start filling out the prompts

**Issue(s)**
* closes https://github.com/HHS/Head-Start-TTADP/issues/100

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
